### DISCUSSION TITLE
Starter Pack

### DISCUSSION THREAD

**Discussion by Bag72** [mealie-recipes/mealie#4710] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

I downloaded Mealie after I found it on Youtube.    Very nice tool but I have one suggestion.   Would it not be possible to have the base system have some recipes in there to start with.   I would guess we are all out scraping the same websites and millions of man hours are being wasted.

### Please provide a concise description of the feature that would resolve your issue.

Can we have a Mealie compliant DB that we could import.  Or is there any users out there with databases they are willing to share.   Any sites out there that can be scraped easily?

### Please consider and list out some caveats or tradeoffs made in your design decision

I understand that there may be copyright issues but humans have been cooking for a very long time.  There must be millions of recipes that would not have any issues.

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by ChipSkylark37** [comment]
Perhaps a sample breakfast, lunch, and dinner could be included by default just for a sample and to show you how to do things for a guided demo / tutorial, on your own instance. 

"Edit this recipe and add a tag, change to show ingredient values, add to a timeline, etc" that would really be a way to showcase the features without just reading through documentation.
